run

```diff
@@ -177,12 +204,13 @@ case $run_command in
 
     # Run watch command in the background
     if [[ ${1:-} == "--watch" ]] || [[ ${1:-} == "-w" ]]; then
-      container_name="yarn-$(date +'%s')"
-      trap 'docker kill ${container_name}' EXIT
+      container_name="${project}-yarn-watch"
+      kill_docker_container "${container_name}"
       docker_run  \
         --name ${container_name}  `# Name the unit so it can be killed` \
         --detach                  `# Run in the background` \
         $yarn_image run watch     `# Use the image for node version 7`
+      trap "kill_docker_container '${container_name}'" EXIT INT TERM
```

Comment:
I was reading that EXIT doesn't always fire from a shell, as it isn't a shell standard. So run could be stopped externally and not fire it off.

But you bringing this up made me look into it again and it is easier/safer not to worry about that unless it comes up.